lib/kaiser/kaiserfile.rb

```diff
@@ -17,6 +18,7 @@ def initialize(filename)
 
       @database = {
         image: 'alpine',
+        platform: '',
```

Comment:
Another part of the change

Reply:
wdyt about setting it to `nil` for a consistency with the other place where we set platform?